.buildkite/pipeline.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
It looks like this is not forced by linter :(

Reply:
Yeah :slightly_frowning_face: I don't know how that linter works.